src/pages/foundation/FoundationFooter.tsx

```diff
@@ -0,0 +1,156 @@
+        { label: t('wallets'), url: '/wallets' },
+        { label: t('journey'), url: '/journey' },
+        { label: t('wiki'), url: 'https://www.wikiwand.com/en/Nervos_Network' },
+        {
+          label: t('media_kit'),
+          url: '/media-kit',
+        },
+      ],
+    },
+    {
+      title: (
+        <StyledLink className={styles.titleLink} href="/developers">
+          {t('developers')}
+        </StyledLink>
+      ),
+      links: [
+        { label: t('documentation'), url: 'https://docs.nervos.org/' },
+        { label: t('github'), url: 'https://github.com/nervosnetwork/' },
+        { label: t('explorer'), url: 'https://explorer.nervos.org/' },
+      ],
+    },
+    {
+      title: t('ecosystem'),
+      links: [
+        { label: t('cryptape'), url: 'https://cryptape.com/' },
+        { label: t('godwoken'), url: 'https://godwoken.com/' },
+        { label: t('nervina_labs'), url: 'https://nervina.io/' },
+        { label: t('tunnel_vision_labs'), url: 'https://tunnelvisionlabs.xyz/' },
+        { label: t('axon'), url: 'https://axonweb3.io/' },
+        { label: t('.bit'), url: 'https://www.did.id/' },
+        { label: t('magickbase'), url: 'https://github.com/Magickbase/' },
+      ],
+    },
+    {
+      title: (
+        <StyledLink className={styles.titleLink} href="/community">
+          {t('community')}
+        </StyledLink>
+      ),
+      links: [
+        { label: t('community_fund_dao'), url: 'https://dao.ckb.community/' },
+        { label: t('nervos_talk_forum'), url: 'https://talk.nervos.org/' },
+        { label: t('rfcs'), url: 'https://github.com/nervosnetwork/rfcs/' },
+        { label: t('ckb_nodes'), url: 'https://nodes.ckb.dev/' },
+      ],
+    },
+    {
+      title: (
+        <StyledLink className={styles.titleLink} href="/learn">
+          {t('learn')}
+        </StyledLink>
+      ),
+      links: [
+        { label: t('knowledge_base'), url: '/knowledge-base' },
+        { label: t('medium'), url: 'https://medium.com/nervosnetwork' },
+        { label: t('youtube'), url: 'https://www.youtube.com/c/NervosNetwork' },
+      ],
+    },
+  ]
+
+  // Todo: the url is temporarily filled with placeholders
+  const medias: MediasProps[] = [
+    {
+      label: <XIcon />,
+      url: 'https://x.com/NervosNetwork',
+    },
+    {
+      label: <DiscordIcon />,
+      url: 'https://discord.gg/FKh8Zzvwqa',
+    },
+    {
+      label: <TelegramIcon />,
+      url: 'https://t.me/NervosNetwork',
+    },
+    {
+      label: <LinkedInIcon />,
+      url: 'https://www.linkedin.com/company/nervos',
+    },
+    {
+      label: <RedditIcon />,
+      url: 'https://www.reddit.com/r/NervosNetwork/',
+    },
+    {
+      label: <YoutubeIcon />,
+      url: 'https://www.youtube.com/c/NervosNetwork',
+    },
+    {
+      label: <TIcon />,
+      url: 'https://talk.nervos.org/',
+    },
+  ]
+
+  return (
+    <div className={clsx(styles.footer, className)} {...divProps}>
```

Comment:
There is a gap between footer and bottom to make the footer not stick to the edge
<img width="837" alt="image" src="https://github.com/Magickbase/nervos-official-website/assets/7271329/3299808b-c3e4-4ce6-b7e9-963b2170479a">

Others are all good